docs/performance.md

```diff
@@ -19,6 +19,21 @@ Next we show the loss curves for Llama 3 8B and Llama 3 70B training with both 1
 ![image](../assets/images/llama3_loss_curves.png)
 
 
+## Llama 3.1 performance numbers
+
+We did a performance measurement on the 405B model released in [LLaMa 3.1](https://llama.meta.com/docs/model-cards-and-prompt-formats/llama3_1). Because the model now is larger, we run on 128 H100 GPUs to test both performance and loss curves. Below is the performance result of 405B model with optimizations we have developed. We do see OOM for 1D even with batch size = 1, so we only tested the 2D case.
+
+
+| Model size | Batch size | Activation checkpointing | WPS | MFU | optimizations |
+| ----- | ----- | ----- | ----- | ----- | ----- |
+| 405B | 2 | full | 118 | 37.1% | None
+| 405B | 2 | full | 177 | 27.77% | FP8
+| 405B | 2 | full | 185 | 29.03% | FP8 + async TP
```

Comment:
I think now they call it Float8 instead of FP8, e.g. see https://dev-discuss.pytorch.org/t/enabling-float8-all-gather-in-fsdp2/2359

Let's follow that.